awesomeCreateApp.js

```diff
@@ -1,11 +1,9 @@
-const child_process = require('child-process-promise')
 const chalk = require('chalk')
 const inquirer = require('inquirer')
 const emoji = require('node-emoji')
 const hooks = require('./hooks')
+const spawnPromise = require('./utils/spawnHelper')
```

Comment:
maybe just `spawn` is better, all the async functions used in this context are using promises 😄